src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -43,6 +43,11 @@
     <TSqlScript Include="Features\Schema\Sql\Tables\*.sql" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
We can remove this section, all new files will be included with lines 26 and 27. 
31diff.sql doesn't seem right.

Reply:
Removed.